$MirOS: ports/misc/mc/patches/patch-edit_edit_h,v 1.3 2006/02/21 13:39:40 tg Exp $
--- edit/edit.h.orig	Mon Jan 27 15:59:50 2003
+++ edit/edit.h	Tue Feb 21 14:04:14 2006
@@ -330,7 +330,7 @@ extern int edit_confirm_save;
 extern int column_highlighting;
 
 /* File names */
-#define EDIT_DIR           PATH_SEP_STR ".mc" PATH_SEP_STR "cedit"
+#define EDIT_DIR           PATH_SEP_STR ".etc" PATH_SEP_STR "mc" PATH_SEP_STR "cedit"
 #define SYNTAX_FILE        EDIT_DIR PATH_SEP_STR "Syntax"
 #define CLIP_FILE          EDIT_DIR PATH_SEP_STR "cooledit.clip"
 #define MACRO_FILE         EDIT_DIR PATH_SEP_STR "cooledit.macros"
